dist/images/install.sh

```diff
@@ -3903,7 +3149,7 @@ if ! sh -c "echo \":$PATH:\" | grep -q \":/usr/local/bin:\""; then
 fi
 
 echo "[Step 6/6] Run network diagnose"
-kubectl ko diagnose all
+docker exec kube-ovn-control-plane kubectl ko diagnose all
```

Comment:
kubectl ko should work in current node.

And kube-ovn-control-plane doesn't exist in a non-kind cluster